scripts/test-runner.sh

```diff
@@ -47,8 +47,13 @@ if [[ $OS_TYPE != "fedora" ]]; then
     $PKG_MAN -y install epel-release
 fi
 
+# TEMPORARY WORKAROUND
+# TODO: Remove when a fixed official beakerlib version is released
+yum -y install https://sumsal.cz/dev/beakerlib-1.11centos-99.fc23.noarch.rpm
```

Comment:
After a few hours of experimenting and debugging I have good news and bad news.

The good news is that for installing the latest version of beakerlib we can use repo `epel-testing` on CentOS and `updates-testing` on Fedora. Both repos contain the fixed version of beakerlib.

The bad news is that the new version fixes a bug in `rlWaitForSocket` function. This function now actually waits for socket to close when `--close` is specified. This would be great, but the way how it checks the socket is horrible:

```
local cmd="netstat -nla | grep -E '$grep_opt' >/dev/null"
```

This thing right here checks if the socket/port is listed in netstat's output, but this includes **remote** ports along with local ones, which is a problem. Many interoperability tests kill the server part, wait for its PID to disappear and then wait for the socket to close. But the nestat's output in this case contains client's socket as well.

Long story short, each `rlRun "rlWaitForSocket 4433 --close"` now takes around 50 seconds to complete, because of this:
```
# netstat -nla
Active Internet connections (servers and established)
Proto Recv-Q Send-Q Local Address           Foreign Address         State   
...
tcp6       0      0 ::1:56226               ::1:4433                TIME_WAIT
```

This means, that many interoperability test now take several to dozens of hours to complete, which is totally useless, especially in Travis. Fix for this issue is not complicated, but it can take another several months to propagate to testing repos. @tomato42, @ep69 any ideas?